.github/workflows/build.yml

```diff
@@ -63,13 +63,14 @@ jobs:
     - name: Bootstrap
       run: |
         sudo apt-get update
-        sudo apt-get --no-install-recommends install -y clang-format-14 clang-tidy-14 shellcheck
+        sudo apt-get --no-install-recommends install -y clang-format-14 clang-tidy-14 shellcheck iwyu
```

Comment:
Should we restrict the version of the `iwyu` tool to a fixed version, similar to how we handle `clang-format` and `clang-tidy` (e.g., `-14`)? This would ensure consistent behavior.

Reply:
It should be fine to use the latest version, because there are no so many options like clang-format and clang-tidy. And the result is either removing or adding, no significant differences between different versions. Thoughts?